webknossos/examples/apply_merger_mode.py

```diff
@@ -0,0 +1,91 @@
+from pathlib import Path
+from shutil import copyfileobj
+from typing import Tuple, cast
+from urllib.request import urlopen
+from zipfile import ZipFile
+
+import fastremap
+
+import webknossos as wk
+
+
+def main() -> None:
+
+    ####################################
+    # Opening a merger mode annotation #
+    ####################################
+
+    nml = wk.Skeleton.load(Path("testdata/annotations/l4_sample__merger_mode.nml"))
+
+    ###############################################
+    # Download and open the corresponding dataset #
+    ###############################################
+
+    copyfileobj(
+        urlopen("https://static.webknossos.org/data/l4_segmentation.zip"),
+        open("testdata/l4_segmentation.zip", "wb"),
+    )
+    ZipFile("testdata/l4_segmentation.zip").extractall("testdata")
+
+    dataset = wk.Dataset.open("testdata/l4_segmentation")
+    in_layer = cast(wk.SegmentationLayer, dataset.get_layer("segmentation"))
+    in_mag1 = in_layer.get_mag("1")
+
+    ###############################
+    # Compute equivalence classes #
+    ###############################
+
+    equiv_classes = [
+        set(
+            in_mag1.read(absolute_offset=node.position, size=(1, 1, 1))[0, 0, 0, 0]
+            for node in graph.nodes
+        )
+        for graph in nml.flattened_graphs()
+    ]
+
+    equiv_map = {}
+    for segment_ids in equiv_classes:
+        base = next(iter(segment_ids))
+        for segment_id in segment_ids:
+            equiv_map[segment_id] = base
```

Comment:
This can be simplified a bit without having `equiv_classes` first. Also `ndarray.item()` is nice for this usecase:
```suggestion
    equivalence_map = {}
    for graph in nml.flattened_graphs():
        base = None
        for node in graph.nodes:
            segment_id = in_mag1.read(absolute_offset=node.position, size=(1, 1, 1)).item()
            if base is None:
                base = segment_id
            equivalence_map[segment_id] = base
```